ontology/yaml/resources/HVAC/entity_types/DOAS.yaml

```diff
@@ -253,6 +253,21 @@ DOAS_SFVSC_EFVSC_HTWHLSTC_CHWVM_HWVM_SFM_FDPSM_ACO2C_SDM2X_EDM2X:
   - SDM2X
   - EDM2X
 
+DOAS_DFVSC_CHWDDC_OAMC_OA:
+  guid: "1a52f54a-6e0f-4565-83cf-90be0d8f4d3c"
+  description: "Single zone DOAS with damper and discharge fan speed control."
+  is_canonical: true
+  implements:
+  - DOAS
+  - DFVSC
+  - DSPM
+  - CHWDDC
+  - OAMC
+  - OA
+  opt_uses:
```

Comment:
do not define these single fields here - check if they are defined on the DOAS level